components/card/card.css

```diff
@@ -0,0 +1,105 @@
+.container-card{
+    margin: 2rem;
+    display: flex;
+    flex-direction: row;
+    gap: 3.125rem;
+    flex-wrap: wrap;
+}
+
+.vertical-card{
+    position: relative;
+    display: flex;
+    flex-direction: column;
+    justify-content: space-between;
+    max-width: 16.25rem;
+    width: 100%; 
+    box-shadow: var(--btn-box-shadow);
+    padding-top: 0.5rem;
+}
+
+.img-container{
+    max-width: 14.375rem; 
+    max-height: 16.875rem;
+    margin: 0.938rem;
+    display: flex;
+    flex-wrap: nowrap;
+    object-fit: cover;
+    
+}
+
+.text_overlay-card {
+    opacity: 0.2;
+}
+
+.text-overlay{
+    display: block;
+    position: absolute;
+    left: 15%;
+    bottom: 50%;
```

Comment:
No specific reason. I am habitual to using %. Sometimes I find it easy to visualize the position of the element. Ex: 50 %. I don't know if it is bad practice.

Reply:
- I can't say it's bad practice but it will affect your responsiveness because I saw some time for positioning are you using rem and % also.
- So it will good if you stick with one unit

Reply:
Thanks Anuj, I will change it to rem.